src/main/java/duke/handler/Duke.java

```diff
@@ -13,7 +12,7 @@
 import java.io.IOException;
 
 /**
- * A class representing the Duke App.
+ * A class manages duke operation.
  */
 public class Duke {
     private static final String FILEPATH = "/Users/jiangyuxin/Documents/sem1/cs2103/duke/data/duke.txt";
```

Comment:
It seems, with the new GUI implementation, that this filepath is now a duplicate of the one in Main.java, and is no longer used. 

If there is any need for the CLI version of Duke to be used, perhaps could consider abstracting the filepath location? (unless the filepath of the CLI and GUI version of Duke is meant to be tested separately)